MekHQ/src/mekhq/campaign/mission/AtBDynamicScenario.java

```diff
@@ -111,11 +149,27 @@ public int getMapY() {
         return getMapSizeY();
     }
     
+    /**
+     * Adds a bot force to this dynamic scenario.
+     * @param botForce
+     * @param forceTemplate
+     */
     public void addBotForce(BotForce botForce, ScenarioForceTemplate forceTemplate) {
         super.addBotForce(botForce);        
         botForceTemplates.put(botForce, forceTemplate);
     }
     
+    /**
+     * Removes a bot force from this dynamic scenario, and its associated template as well.
+     */
+    public void removeBotForce(int x) {
+        BotForce botToRemove = botForces.get(x);
```

Comment:
Likely going to want to validate `x` (is this an ID or index?) and/or `botToRemove`.

Reply:
Added safety check.